Source/Renderer/Texture.js

```diff
@@ -164,10 +166,26 @@ define([
                 // Source: ImageData, HTMLImageElement, HTMLCanvasElement, or HTMLVideoElement
                 gl.texImage2D(textureTarget, 0, internalFormat, pixelFormat, pixelDatatype, source);
             }
+            gl.bindTexture(textureTarget, null);
         } else {
+            // Initialize the pixels to zero to make sure that the texture was loaded, this is a workaround for chrome and may not always be necessary
+            var framebuffer = gl.createFramebuffer();
+            gl.bindFramebuffer(framebufferTarget, framebuffer);
+
+            texture = gl.createTexture();
+            gl.activeTexture(gl.TEXTURE0);
+            gl.bindTexture(textureTarget, texture);
             gl.texImage2D(textureTarget, 0, internalFormat, width, height, 0, pixelFormat, pixelDatatype, null);
+            gl.framebufferTexture2D(framebufferTarget, gl.COLOR_ATTACHMENT0, textureTarget, texture, 0);
+
+            if(pixelFormat === gl.RGBA) {
```

Comment:
Does this only happen for `RGBA` textures? What about `RGB` textures? I doubt we would have the problem with any of the depth formats since they are only used as render targets.